tdex/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.8"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: tdex_caddy_1
+      APP_PORT: 8080
+  
+  tdexd:
+    image: ghcr.io/tdex-network/tdexd:v0.8.19@sha256:5a2c0db9f74812e67aa98f50f396e81edd23ef49cb6797b084027ba6124db707
+    tty: true
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/tdex-data:/home/tdex/.tdex-daemon
+    environment:
+      TDEX_LOG_LEVEL: 5
+      TDEX_EXPLORER_ENDPOINT: https://liquid.network/liquid/api
+      TDEX_OPERATOR_LISTENING_PORT: 9090
+      TDEX_TRADE_LISTENING_PORT:  9090
+      TDEX_NO_OPERATOR_TLS: "true"
+      # these vars are used to generate the connect URL
+      # we assume to always show the external endpoint
+      # reachable remotely by other dashboards via Tor
+      TDEX_CONNECT_ADDR: "$APP_HIDDEN_SERVICE"
+      TDEX_CONNECT_PROTO: http
```

Comment:
@tiero As the dashboard now automatically detects the correct connection url. Potentially these 2 variables are no longer needed?

Reply:
These variables are needed for the daemon to also generate a html page used to retrieve the TDEX CONNECT URL  (it should be visibile navigating to `/api/` with the browser) 

Useful if users wants to connect from outside using the tdex dashboard as dekstop app.

Reply:
> we can grab the hostname for the daemon specifically and feed that in?

What do you mean here? how you can "introspect" the hostname, with a bash script or something?

Reply:
> Tor Hidden Service url for the daemon specifically. ie xyz.onion:9090 

How could the user know about the two difference onion services that are running? ie. one to use to navigate to the dashaboard and the other to give to traders?

Reply:
I know Tor is terrible, but it's the only way for TDEX Daemon operators to expose the public trade API and allow TDEX App and other clients to do NAT traversal and reach the daemon running on their Umbrel.

An alternative would be to have services such as ngrok?

Reply:
Yh, unfortunately at this time it's a little tricky in terms of remote access. Potentially Tailscale is another option (that we have in the app store), but that also has a few challenges.

Reply:
I bumped the dashboard with this new ENV VAR to feed the hidden service.

I created APP_TDEX_TRADER_HIDDEN_SERVICE, hope this works.

Can you help me to create the exports.sh file?